### PR TITLE
developers/ page update

### LINKED ISSUE #468: Implement designs for developers/ page
I'm going to try and implement the new design for the developers page. This should be fairly straightforward. If not, someone can adjust as necessary.

### LINKED ISSUE #471: add redirect for /developer
Following up from [missing the developer hub the other day](https://github.com/18F/foia-hub/issues/462), I'd recommend that `https://foia.18f.us/developer` resolve to `https://foia.18f.us/developers`

### PR DESCRIPTION
**Pull request by khandelwal**
The developers/ page is now updated with the new designs. (fixes #468)

I also aliased developers/ and developer/ to go to the same page, as there is some discussion about this. (fixes #471)

### CODE REVIEW

File: foia_hub/scripts/load_agency_contacts.py [hunk 1]
```diff
@@ -8,10 +8,10 @@
 import yaml
 from glob import iglob
 import django
-django.setup()
 
 from foia_hub.models import Agency, Office, Stats, ReadingRoomUrls
 
+django.setup()
```
Comment (khandelwal) [inline foia_hub/scripts/load_agency_contacts.py]: This change is related to the fact that the new flake8 likes all the imports up top.

File: foia_hub/templates/about.html [hunk 2]
```diff
@@ -68,7 +68,7 @@
 
             <h2>More about this project</h2>
 
-            <p>Our work is one of a number of <a href="http://www.whitehouse.gov/sites/default/files/docs/us_national_action_plan_6p.pdf">commitments</a> towards modernizing the Freedom of Information Act. We are currently in the <a href="https://18f.gsa.gov/dashboard/stages/">alpha</a> stage of development. Developers can find additional technical information <a href="/developers">here.</a></p>
+            <p>Our work is one of a number of <a href="http://www.whitehouse.gov/sites/default/files/docs/us_national_action_plan_6p.pdf">commitments</a> towards modernizing the Freedom of Information Act. We are currently in the <a href="https://18f.gsa.gov/dashboard/stages/">alpha</a> stage of development. Developers can find additional technical information <a href="{{url('developers')}}">here.</a></p>
```
Comment (khandelwal) [inline foia_hub/templates/about.html]: Trying to avoid hardcoding the urls in the templates.

File: foia_hub/templates/developers.html [hunk 3]
```diff
@@ -1,99 +1,116 @@
...
+            <h1><em>Developers</em></h1>
 
-<h1> Developers </h1>
+            <p>
+            This effort is a work in progress by <a href=https://18f.gsa.gov/"">18F</a>.
+            However, we welcome contributions, suggestions and comments by everyone.
+            </p>
 
-<p>
-Hello!
```
Comment (khandelwal) [inline foia_hub/templates/developers.html]: This is not the same tone we have elsewhere.

**Comment by khandelwal** [comment]
screen shot 2015-02-09 at 3 56 07 pm

File: foia_hub/urls.py [hunk 4]
```diff
@@ -21,6 +21,9 @@
     url(r'^agencies/?$', agencies, name='agencies'),
     url(r'^developers/?$', TemplateView.as_view(
         template_name="developers.html"), name='developers'),
+    # Add in the singular version as well.
+    url(r'^developer/?$', TemplateView.as_view(
```
Comment (ramirezg) [inline foia_hub/urls.py]: Could we do one url but set up the regex like this `'^developer[s]?/?$'` ?

Comment (khandelwal) [inline foia_hub/urls.py]: I feel like my way is clearer.

**Comment by ramirezg** [comment]
Read through the text and tested it locally. Looks good to me.

**Pull request merged**